metersink/lib.py

```diff
@@ -41,8 +35,8 @@ def get_config(path):
 
 
 def get_config_section(_config, section=None):
```

Comment:
What is the type of `_config`? What is the point of the (default!) case that `section` is `None`?

Reply:
The question was not meant literally. What I wanted to say was: I think this function needs better documentation or at least type annotation.

Reply:
The section argument is now required. the default path has been removed now.